CourtBallGame/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from court.models import Court
+
+MAX_LEN_BALL_GAME_STR = 100
+
+
+class BallGame(models.TextChoices):
```

Comment:
This enum is already defined in `player.models`, after rebasing you should be able to import it